### DISCUSSION TITLE
How to Filter All installed Ids?

### DISCUSSION THREAD

**Discussion by ghost1372** [microsoft/winget-cli#1204] [Q&A] [answered]
How we can filter All installed Packages Ids?
Currently, the following command displays all information

`winget list`

Is there any command to display only one field (id)?

**Comment by denelon** [comment, marked as answer]
@ghost1372 we don't currently have support for what you are asking for. Be sure to add your 👍 on [Enhancements to list command](https://github.com/microsoft/winget-cli/issues/1155)
